binding/python/README.md

```diff
@@ -1,11 +1,90 @@
-# Prerequisites
+# Rhino Speech-to-Intent Engine
 
-Python 3 is required to use the binding and run its accompanying unit tests.
+Made in Vancouver, Canada by [Picovoice](https://picovoice.ai)
 
-# Running Unit Tests
+Rhino is Picovoice's Speech-to-Intent engine. It directly infers intent from spoken commands within a given context of
+interest, in real-time. For example, given a spoken command *"Can I have a small double-shot espresso with a lot of sugar
+ and some milk"*, Rhino infers that the user wants to order a drink with these specifications:
 
-Using command line (from the root of the repository)
+```json
+{
+  "type": "espresso",
+  "size": "small",
+  "numberOfShots": "2",
+  "sugar": "a lot",
+  "milk": "some"
+}
+```
+
+Rhino is:
+
+* using deep neural networks trained in real-world environments.
+* compact and computationally-efficient, making it perfect for IoT.
+* self-service. Developers and designers can train custom models using [Picovoice Console](https://picovoice.ai/console/).
+
+## Compatibility
+
+- Python 3
+- Runs on Linux (x86_64), Mac (x86_64), Windows (x86_64), Raspberry Pi (all variants), and BeagleBone.
+
+## Installation
 
 ```bash
-python3 binding/python/test_rhino.py
+pip3 install pvrhino
+```
+
+## Usage
+
+Create an instance of the engine
+
+```python
+import pvrhino
+
+handle = pvrhino.create(context_path='/absolute/path/to/context')
+```
+
+Where `context_path` is the absolute path to Speech-to-Intent context created either using
+[Picovoice Console](https://picovoice.ai/console/) or one of the default contexts available on Rhino's GitHub repository.
+
+The sensitivity of the engine can be tuned using the `sensitivity` parameter. It is a floating point number within
+[0, 1]. A higher sensitivity value results in fewer misses at the cost of (potentially) increasing the erroneous
+inference rate.
+
+```python
+import pvrhino
+
+handle = pvrhino.create(context_path='/absolute/path/to/context', sensitivity=0.)
```

Comment:
Use a value not far from 0.5 to encourage users to start with default and move in increments away from it instead of going full 0/1

Reply:
makes sense.